docs/content/stable/explore/observability/active-session-history.md

```diff
@@ -80,7 +80,7 @@ This view provides a list of wait events and their metadata. The columns of the
 | wait_event_aux | text | Additional information for the wait event. For example, tablet ID for TServer wait events. |
 | top_level_node_id | UUID | 16-byte TServer UUID of the YSQL/YCQL node where the query is being executed. |
 | query_id | bigint | Query ID as seen on the `/statements` endpoint. This can be used to join with [pg_stat_statements](../../query-1-performance/pg-stat-statements/)/[ycql_stat_statements](../../query-1-performance/ycql-stat-statements/). It is set as a known constant for background activities. For example, _flush_ is 2, _compaction_ is 3, and so on. |
```

Comment:
Instead of listing : 

`set as a known constant for background activities. For example, _flush_ is 2, _compaction_ is 3, and so on ` . We can link the `Constant query identifiers` section and ask users to refer to that.